sample/.github/workflows/test.yml

```
@@ -0,0 +1,50 @@
+name: Test
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+
```

Shall we put the same thing as in templates?
```suggestion
  push:
    branches-ignore:
      - develop
      - 'release/**'
      - main
```

Or I guess you will rebase later? 🤔

---

@lydiasama We already have `pull_request` to verify the template script. Add `push` event is redundant here and will cause conflict between these events when pushing sample project changes.
- push:  https://github.com/nimblehq/kmm-templates/actions/runs/5875799533/job/15932720203 ✅ 
- pull_request: https://github.com/nimblehq/kmm-templates/actions/runs/5875800008 ❌

---

One `pull_request` event is enough https://github.com/nimblehq/kmm-templates/actions/runs/5876269164?pr=28 ✅